src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -50,7 +54,7 @@ public class ResourceManagerStringLocalizer : IStringLocalizer
         /// </summary>
         public ResourceManagerStringLocalizer(
             ResourceManager resourceManager,
-            AssemblyWrapper resourceAssemblyWrapper,
+            IResourceStreamManager resourceStreamManager,
```

Comment:
Changing this would be breaking. But you should be able to convert `resourceAssemblyWrapper` to `AssemblyResourceStreamManager` and introduce a new ctor.